build.sbt

```diff
@@ -7,7 +7,7 @@ name := "atom-maker-lib"
 
 lazy val baseSettings = Seq(
   organization := "com.gu",
-  scalaVersion := "2.11.11",
+  scalaVersion := "2.11.12",
```

Comment:
`2.12` soon @regiskuckaertz ?

Reply:
🤦‍♂️  of course, will add that!

Reply:
this is actually only used in atom-workshop which is still running on 2.11. I can look into it eventually ✌️